### PR TITLE
Added member-list w/ names+pics

### PR DESCRIPTION
**Pull request by DanielSosa521**

### CODE REVIEW

**Comment by sidharthNair** [comment]
Looks good to me; it may be a good idea to use an ant design component later because they may have more features (for example https://ant.design/components/image/ have a zoom in feature)

File: src/components/MemberItem.jsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react'
+import "../pages/css/MemberItem.css"
+
+class MemberItem extends React.Component {
+    constructor(props) {
+        super(props)
+    }
+    render() {
+        return (
+            <div class="member-item">
```
Comment (Ti-tanium) [inline src/components/MemberItem.jsx:10]: should be `className` in react instead of class.

File: src/pages/Home.jsx [hunk 2]
```diff
@@ -1,10 +1,32 @@
...
+import MemberItem from "../components/MemberItem";
+import "./css/Home.css"
 
 export default function Home() {
+  const mediaPrefix = "/media/members/"
   return (
     <div>
       {" "}
       <img style={{ with: 20, height: 50, float: "left" }} src="logo.png" />
+      <br/>
```
Comment (Ti-tanium) [inline src/pages/Home.jsx:11]: it's better to use css margin bottom property than use `<br/>` for spacing

**Review by Ti-tanium — CHANGES_REQUESTED** [review]
Another piece of advice: The list can also use a UI component from `Ant design` as well, so you don't need to write the styles yourself.

**Review by Ti-tanium — APPROVED** [review]

**Pull request merged**